File: Editor/src/main/java/org/turbomedia/turboedit/editor/panes/render_queue/RenderQueuePane.java
package org.turbomedia.turboedit.editor.panes.render_queue;

import javafx.scene.control.TabPane;
import org.turbomedia.turboedit.editor.misc.PreferencesFile;

public class RenderQueuePane extends TabPane {
    public RenderQueuePane() {
        setMaxWidth(Double.MAX_VALUE);
        setMaxHeight(Double.MAX_VALUE);
        setTabClosingPolicy(TabClosingPolicy.UNAVAILABLE);

        for (var server : PreferencesFile.CURRENT_PREFERENCES.renderServers) {
            getTabs().add(new ServerQueue(server));
        }
    }
}

File: Editor/src/main/kotlin/turboedit/editor/panes/render_queue/RenderQueuePane.kt
package turboedit.editor.panes.render_queue

import javafx.scene.control.TabPane
import turboedit.editor.misc.PreferencesFile

class RenderQueuePane : TabPane() {
    init {
        maxWidth = Double.Companion.MAX_VALUE
        maxHeight = Double.Companion.MAX_VALUE
        tabClosingPolicy = TabClosingPolicy.UNAVAILABLE

        for (server in PreferencesFile.CURRENT_PREFERENCES!!.renderServers) {
            tabs.add(ServerQueue(server))
        }
    }
}
